build/release.mk

```diff
@@ -45,12 +45,6 @@ release/fix-archive:
 	@echo "===> $(INTEGRATION) === [release/fix-archive] fixing zip archives internal structure"
 	@bash $(CURDIR)/build/windows/fix_archives.sh $(CURDIR)
 
-.PHONY : release/sign/nix
-release/sign/nix:
-	@echo "===> $(INTEGRATION) === [release/sign] signing packages"
-	@bash $(CURDIR)/build/nix/sign.sh
```

Comment:
we can also garbage collect the key from the secrets repo if they are not managed by org

Reply:
Omw